Modules/Typescript/Patterns/controller.ts

```diff
@@ -0,0 +1,150 @@
+import { EventEnum, canvasId } from './utils';
+import { TextInputsView } from './text-inputs-view';
+import { SingleEuroModel } from './single-euro-model';
+import { SlidersView } from './sliders-view';
+import * as simpleData from './data.json';
+
+export class Controller {
```

Comment:
You should separate rendering logic to the view and leave only the user input handling in the controller